python/private/pypi/dependency_resolver/dependency_resolver.py

```diff
@@ -27,9 +27,9 @@
 
 from python.runfiles import runfiles
 
-# Replace the os.replace function with shutil.copy to work around os.replace not being able to
+# Replace the os.replace function with shutil.move to work around os.replace not being able to
 # replace or move files across filesystems.
-os.replace = shutil.copy
+os.replace = shutil.move
```

Comment:
FYI, I am looking at the PR that made the change previously (#1053) and I can't see why one wouldn't want the `shutil.move`. I also can't see looking at the source of `shutil.move` if it would be problematic, it will also try to fallback to `shutil.copy2`, so I think the `shutil.move` is a robust solution here.